sspl_test/sspl-test.spec

```diff
@@ -78,6 +78,18 @@ else
     #touch ${SSPL_DIR}/sspl_test/keep_flask
 fi
 
+# Check and install required coverage version
+cov=`pip3.6 freeze | grep coverage`
+if [[ -n $cov ]]; then
+    ver=${cov##*=}
+    if [[ "$ver" != "5.5" ]]; then
+        pip3 uninstall -y coverage
+        pip3 install coverage==5.5
```

Comment:
coverage pkg needs to be part of cortx-prerequisites pkg maintained by RE. 

@ujjwalpl & @shailesh-vaidya cortx-sspl-test needs this 'coverage' pkg in dev/test env where code coverage needs to generated & collected, is it ok to keep it cortx-prerequisites, even if it is needed in dev/qa env only ?

Reply:
@sumedhak27 follow up with Shailesh tomorrow

Reply:
okay sure..

Reply:
@cdeshmukh @sumedhak27  I guess coverage package is build time dependency and not run time dependency (Not required for deployment) . We should only add run time dependencies in `cortx-prereq` packages to keep third party packages less. 
You can use `coverage` package during build time without any issues. Let me know if you face any problem there.

Reply:
@shailesh-vaidya Its cortx-sspl-test rpm post install requirement, its not bundled in rpm during build. 
Now cortx-sspl-test itself is optional, so it doesnt affect deployment. Though for running sspl self-tests after deployment this cortx-sspl-test package is needed. 
At run time coverage module searched for, if tests started with argument to generate code coverage.

So i still see it as run time dependency, unless am thinking something wrong, please suggest.

Reply:
@sumedhak27 Spoke to @ujjwalpl about this. First it doesnt need to be part of cortx-prerequisites given its not needed by a must cortx pkg. Since needed by optional sspl-test package, its deployment should relate to that.

Secondly, any dependent package installation should happen through 'Requires' attribute if dependent pkg is an rpm. Else such installation should not happen through rpm post-install stage, but through mini provisioner interface. So this coverage & even flask package installation should happen during mini-provisioner 'test' invocation, basically check for pkg & install it. This would also mean, any test execution should happen through mini provisioner interface only going forward & not through direct sspl test scripts like its done today.

Reply:
THis was discussed again. 
1. (preferred) If not possible to pull dependent pkg, through rpm 'Requires' attribute, manually install dependent packages before sspl-test rpm installation & update the instructions in wiki with these steps.
Mini provisioner test interface should check for these dependent pkg installation in validate() and fail listing these pkgs.

2. Check & Install dependent packages in mini provisioner interface 'test' before tests are performed even
@sumedhak27

Reply:
so If validator test fails in 'sspl_setup test' for coverage, then should we install the pkg?

Reply:
added Validation and installation of coverage.py in sspl_test mini provisioner's validate phase.